apps/api/v2/src/ee/bookings/controllers/bookings.controller.e2e-spec.ts

```diff
@@ -0,0 +1,205 @@
+      return request(app.getHttpServer())
+        .get(`/api/v2/ee/bookings/${createdBooking.uid}`)
+        .then((response) => {
+          const responseBody: ApiSuccessResponse<Awaited<ReturnType<typeof getBookingInfo>>> = response.body;
+          const { bookingInfo } = responseBody.data;
+
+          expect(responseBody.status).toEqual(SUCCESS_STATUS);
+          expect(responseBody.data).toBeDefined();
+          expect(bookingInfo?.id).toBeDefined();
+          expect(bookingInfo?.uid).toBeDefined();
+          expect(bookingInfo?.id).toEqual(createdBooking.id);
+          expect(bookingInfo?.uid).toEqual(createdBooking.uid);
+          expect(bookingInfo?.eventTypeId).toEqual(createdBooking.eventTypeId);
+          expect(bookingInfo?.startTime).toEqual(createdBooking.startTime);
+        });
+    });
+
+    it("should create a recurring booking", async () => {
+      const bookingStart = "2023-05-25T09:30:00.000Z";
+      const bookingEnd = "2023-05-25T10:30:00.000Z";
+      const bookingEventTypeId = 7;
+      const bookingTimeZone = "Europe/Londom";
+      const bookingLanguage = "en";
+      const bookingHashedLink = "";
+      const bookingRecurringCount = 5;
+      const currentBookingRecurringIndex = 0;
+
+      const body = {
+        start: bookingStart,
+        end: bookingEnd,
+        eventTypeId: bookingEventTypeId,
+        timeZone: bookingTimeZone,
+        language: bookingLanguage,
+        metadata: {},
+        hashedLink: bookingHashedLink,
+        recurringCount: bookingRecurringCount,
+        currentRecurringIndex: currentBookingRecurringIndex,
+      };
+
+      return request(app.getHttpServer())
+        .post("/api/v2/ee/bookings/reccuring")
+        .send(body)
+        .expect(201)
+        .then((response) => {
+          const responseBody: ApiResponse<Awaited<ReturnType<typeof handleNewRecurringBooking>>> =
+            response.body;
+
+          expect(responseBody.status).toEqual("recurring");
+        });
+    });
+
+    it("should create an instant booking", async () => {
+      const bookingStart = "2023-05-25T09:30:00.000Z";
+      const bookingEnd = "2023-05-25T10:30:00.000Z";
+      const bookingEventTypeId = 7;
+      const bookingTimeZone = "Europe/Londom";
+      const bookingLanguage = "en";
+      const bookingHashedLink = "";
+
+      const body = {
+        start: bookingStart,
+        end: bookingEnd,
+        eventTypeId: bookingEventTypeId,
+        timeZone: bookingTimeZone,
+        language: bookingLanguage,
+        metadata: {},
+        hashedLink: bookingHashedLink,
+      };
+
+      return request(app.getHttpServer())
+        .post("/api/v2/ee/bookings/instant")
+        .send(body)
+        .expect(201)
+        .then((response) => {
+          const responseBody: ApiResponse<Awaited<ReturnType<typeof handleInstantMeeting>>> = response.body;
+
+          expect(responseBody.status).toEqual("instant");
+        });
+    });
+  });
+});
```

Comment:
We also need to delete all the bookings that been created - you need to create "BookingsRepositoryFixture" just like "UserRepositoryFixture" and can add function that deletes all bookings made to a user aka in this case of "bookings-controller-e2e@api.com" and then add "afterAll" that will call "BookingsRepositoryFixture.deleteAllUserBookings("bookings-controller-e2e@api.com") to do the cleanup